reference/sites.v3.yml

```diff
@@ -210,7 +221,7 @@ paths:
   '/stores/{store_hash}/v3/sites/{site_id}':
     get:
       summary: Get a Site
```

Comment:
We need to update response example for GET site. It now returns all the different URL types for given site :)